gp-media-library/gpml-acf-user-image-field.php

```diff
@@ -44,6 +44,31 @@ function update_user_image_field( $user_id, $feed, $entry ) {
 				$value         = array_merge( $current_value, $value );
 			}
 
+			// Check for uploaded files in the $_POST data, we may need to prevent the ACF field from being set to empty.
+			$raw_json = $_POST['gform_uploaded_files'] ?? '';
+			if ( empty( $value ) && $raw_json ) {
+				$clean_json = stripslashes( $raw_json );
+
+				$uploaded_files_array = json_decode( $clean_json, true );
+
+				$field_id = $this->_args['field_id'];
+				$key      = 'input_' . $field_id;
+
+				if ( isset( $uploaded_files_array[ $key ][0]['uploaded_filename'] ) ) {
```

Comment:
@saifsultanc does this need to work with single file upload fields as well?

If so, you'll likely need to do a check on `$_FILES` if it's a single file upload field. You can check the field type like so: `$field->multipleFiles`

Here's an example of `$_FILES` with a file for a single file upload field.


<img width="364" alt="image" src="https://github.com/user-attachments/assets/eccf1d87-8270-44ea-bbe2-c2fceb351269" />